src/main/java/org/checkerframework/specimin/SpeciminRunner.java

```diff
@@ -110,6 +110,9 @@ public static void performMinimization(
     SourceRoot sourceRoot = new SourceRoot(Path.of(root));
     sourceRoot.tryToParse();
     for (CompilationUnit compilationUnit : sourceRoot.getCompilationUnits()) {
+      // Caution: Calling .get() on an Optional without checking if it's present.
```

Comment:
This `.get()` will only fail if the Java source file of the related compilation unit is not available, such as in the case of a jar file. 

We are iterating through compilation units based on a `SourceRoot`. According to this [documentation](https://javadoc.io/static/com.github.javaparser/javaparser-core/3.4.3/index.html?com/github/javaparser/utils/SourceRoot.html), the above case will not happen, as a `SourceRoot` is:

> A collection of Java source files located in one directory and its subdirectories on the file system. Files can be parsed and written back one by one or all together.